src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -28,20 +29,33 @@ export const ARShowScreen = ({ navigation, route }) => {
     setIsLoading(false);
   };
 
+  const takeScreenshot = useCallback(async () => {
+    const fileName = 'AugmentedReality_' + Date.now().toString();
+
+    try {
+      await arSceneRef.current._takeScreenshot(fileName, true);
```

Comment:
this works good on my device but i wonder if we can have some visual feedback for users to know, that a screenshot was taken successfully? some "screen blinking/opacity/..." or a short message?

Reply:
I planned to do this. I planned to make a flash on the screen.

Reply:
https://user-images.githubusercontent.com/11755668/179163219-7a77fe69-b968-40a6-a04c-8ed599a17ec8.mov

Reply:
nice!